instrumentation/ruby_kafka/lib/opentelemetry/instrumentation/ruby_kafka/instrumentation.rb

```diff
@@ -10,17 +10,23 @@ module RubyKafka
       # The Instrumentation class contains logic to detect and install the
       # KafkaRuby instrumentation
       class Instrumentation < OpenTelemetry::Instrumentation::Base
+        MINIMUM_VERSION = Gem::Version.new('0.7.0')
+
         install do |_config|
           require_patches
           patch
         end
 
         present do
-          defined?(::Kafka)
+          defined?(::Kafka) && gem_version >= MINIMUM_VERSION
```

Comment:
There is a separate hook where this check would be more appropriate: 
https://github.com/open-telemetry/opentelemetry-ruby/blob/b61ea9601accfa2ad92f3c791fb16cc5221cc9b4/api/lib/opentelemetry/instrumentation/base.rb#L122-L130

Reply:
This is done